lib/marc_links.rb

```diff
@@ -37,7 +37,7 @@ def as_h
         link_title:,
         additional_text:,
         material_type: field['3'],
-        note: field['z'],
+        note: subzs,
```

Comment:
Would we want this joined, or is it better to leave it as structured data (and rename to `notes:`)?

Reply:
I'm not sure; I don't think we have any other use cases for this data yet, and the concatenated field is type-compatible with the existing data 🤷‍♂️